pkg/volume/git_repo/git_repo.go

```diff
@@ -190,7 +195,7 @@ func (b *gitRepoVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 		return err
 	}
 
-	args := []string{"clone", b.source}
+	args := []string{"clone", "--", b.source}
```

Comment:
is this supported in all git versions we care about?

Reply:
AFAICT it's been supported since 1.0